gravitee-apim-console-webui/src/portal/top-bar/portal-top-bar.component.html

```diff
@@ -15,5 +15,46 @@
     limitations under the License.
 
 -->
-<portal-header title="Top Bar" subtitle="Customize the Developer Portal top bar" />
-<router-outlet></router-outlet>
+<portal-header title="Manage your top bar sections" subtitle="Manage and edit your Developer Portal pages with Markdown" />
```

Comment:
According to [figma](https://www.figma.com/design/eAS6p0kITPnEI0VPbmATi3/Developer-Portal?node-id=25137-2935&t=WF533tolcSA0rdpt-1), Top bar should be renamed to Navigation
```suggestion
<portal-header title="Manage your navigation" subtitle="Manage and edit your Developer Portal pages with Markdown" />